ts/store/reducers/entities/messages/messagesById.ts

```diff
@@ -5,92 +5,11 @@
  */
 
 import * as pot from "italia-ts-commons/lib/pot";
-import { getType } from "typesafe-actions";
 
 import { CreatedMessageWithContentAndAttachments } from "../../../../../definitions/backend/CreatedMessageWithContentAndAttachments";
 import { CreatedMessageWithoutContent } from "../../../../../definitions/backend/CreatedMessageWithoutContent";
-import {
-  DEPRECATED_loadMessage,
-  removeMessages
-} from "../../../actions/messages";
-import { clearCache } from "../../../actions/profile";
-import { Action } from "../../../actions/types";
-import { GlobalState } from "../../types";
 
 export type MessageState = {
```

Comment:
Why we are removing the reducer but still using the state?

Reply:
Because `MessageState` is a local info used by the pagination migration procedure. Before pagination the archived and read state of a message was local to the device. After moving to pagination these attributes has been moved to the backend.  When you first update to a non-paginated release of IO to a paginated one we upsert these info to the backend, and then clear the store. Until we have users that have "old" version of the app running we should support this feature.